### ISSUE TITLE
Use alternative German main menu buttons

### ISSUE THREAD

**Issue by zenseii** [ihhub/fheroes2#5173]
### Your question

I noticed that there is another set of assets for German buttons in the main menu that get loaded first when loading the main menu and can be seen during the fading animation from high score to main menu.

Maked this as a question, even though I think the buttons in you can see in the fading look better than the ones used in fheroes2 now, because I would like to hear some opinions on this. I haven't looked at where in the resources these files are located.

Had to slow down after exiting the high scores. They can be seen at 14 seconds in the clip.

https://user-images.githubusercontent.com/12501091/160217731-3b19b114-8142-4bd0-b5cb-93bb373f7d59.mp4

**Comment by ihhub** [comment]
Hi @zenseii , many resources related to non-English versions of the game have background with different buttons. It's applicable not only for Main Menu but for other dialogs as well. The biggest issue here is that a button must have at least 2 images: released state and pressed state and the main image (background) has only state.

**Comment by zenseii** [comment]
Yes, the buttons will generally need images for released and pressed state, and for the main menu it only needs an additional highlighting when hovering over the buttons. There is no pressed state and released state.
When I looked through the assets I found this alternative menu to be in the expansion assets' `heroes2x.agg` within `heroes` container. The intention seems to have been to provide a visually improved main menu. Not sure if the assets include a highlighted asset for the buttons.

Here is a png conversion
![000](https://user-images.githubusercontent.com/12501091/162721784-1f85f1b8-ac06-4b85-bc3d-6ff274af3e16.png)

As for highlightng you might just need a yellow border around it similar to how is done with the French assets.
